app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit/Tab/Alerts.php

```diff
@@ -44,15 +44,15 @@ protected function _prepareLayout()
 
         if ($alertPriceAllow) {
             $accordion->addItem('price', [
-                'title'     => Mage::helper('adminhtml')->__('Price alert subscription was saved.'),
-                'content'   => $this->getLayout()->createBlock('adminhtml/catalog_product_edit_tab_alerts_price')->toHtml() . '<br />',
+                'title'     => Mage::helper('adminhtml')->__('Product price alert'),
```

Comment:
1. This should be changed to "Price Alerts". Using Product as prefix is like an abuse in my opinion
2. It needs a row for translation in Mage_Catalog.csv

Reply:
No. Don't add new translation for an useless thing.